components/esp_lcd_touch_gt911/idf_component.yml

```
@@ -0,0 +1,6 @@
+version: "0.0.1"
+description: ESP LCD Touch GT911 - touch controller GT911
+url: https://github.com/espressif/esp-bsp/tree/master/components/esp_lcd_touch_gt911
+dependencies:
+  esp_lcd_touch:
+    version: ">=0.0.1"
```

Applies to all `.yml` files

Since we will probably break the API in version 1.0.0, let's define dependency like this
```suggestion
    version: "^0.9"
```